<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>New_User</class>
 <widget class="QDialog" name="New_User">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>301</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Имя пользователя</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="user_name">
      <property name="maxLength">
       <number>1000</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>248</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Степень активности:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="activ">
      <item>
       <property name="text">
        <string>Минимальная </string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Низкая </string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Умеренная </string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Высокая</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Предельная</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>207</width>
     <height>59</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Возраст:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="age">
        <property name="maxLength">
         <number>2</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Пол:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="men">
        <property name="text">
         <string>Мужской</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="women">
        <property name="text">
         <string>Женский</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>150</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Вес:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="weight"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Рост:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="height"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="reason">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Поддержание веса</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Увеличение массы тела</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Снижение массы тела</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Цель:</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>0</y>
     <width>21</width>
     <height>301</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Создать</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>20</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>260</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="VIBOR">
      <property name="text">
       <string>Выбрать</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="Delete_Button">
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
